SU2_PY/SU2/io/historyMap.py

```diff
@@ -35,50 +35,35 @@
                   'GROUP': 'FLOW_COEFF',
                   'HEADER': 'Avg_Enthalpy',
                   'TYPE': 'COEFFICIENT'},
- 'AVG_MACH': {'DESCRIPTION': 'Total average mach number on all markers set in '
```

Comment:
Did you change this file manually, or did you use a script to do so?
Edit: ... well automatically as it says in the description (using updateHistoryMap.py)